cppapi/client/filedatabase.cpp

```diff
@@ -1612,8 +1635,8 @@ CORBA::Any*   FileDatabase :: DbGetClassProperty(CORBA::Any& send)
 	{
 		if (equalsIgnoreCase((*data_in)[0].in(), m_server.classes[i]->name))
 		{
-			// m_server.classes[i] e' la classe che cerchiamo
-			for (unsigned int j = 1; j < (*data_in).length(); j++)  // in 0 c'e' il nome della classe e poi a seguire le proprieta
+			// m_server.classes[i] is the class we are looking for
+			for (unsigned int j = 1; j < (*data_in).length(); j++)  // at index 0 is the name of the class, property names are following
 			{
 				unsigned long nb_prop_defined = m_server.classes[i]->properties.size();
```

Comment:
Consider adding a variable that would hold property name for the current loop iteration.
```cpp
auto const &prop_name = (*data_in)[j];
```
This `(*data_in)[j]` is repeated three times below. Not only this is unreadable and hard to maintain but also was the source of this bug.